### PR TITLE
Add documentation for changes in config.velocity in v9

### LINKED ISSUE #62: Better documentation needed to migrate velocity props from v8 to v9
## 🐛 Bug Report

The Velocity prop has changed for v9 and the changelog does not mention it

## To Reproduce

A value of `1` in v8 will need to be changed to `0.001` in v9. 

## Expected behavior

This should be in this guide:  https://aleclarson.github.io/react-spring/v9/breaking-changes/

Change to library is here: https://github.com/pmndrs/react-spring/commit/d5ad40fb9f6dccdb7f28bd62b0a5455b1a6a74cf

### PR DESCRIPTION
**Pull request by swashata**
Closes #62

### CODE REVIEW

File: src/pages/common/configs.mdx [hunk 1]
```diff
@@ -30,7 +30,7 @@ And we've added the following properties `frequency`, `damping`, `round`, `bounc
...
 | clamp        | false     | when true, stops the spring once it overshoots its boundaries                                                                                                                                                     |
 | precision    | 0.01      | precision                                                                                                                                                                                                         |
-| velocity     | 0         | initial velocity                                                                                                                                                                                                  |
+| velocity     | 0         | initial velocity. Coming from v8, you need to multiply the value by `0.001` in v9. So a value of `velocity: 5` becomes `velocity: 0.005`.                                                                         |
```
Comment (joshuaellis) [inline src/pages/common/configs.mdx]: See comment on issue.

**Review by joshuaellis — CHANGES_REQUESTED** [review]

File: src/pages/common/configs.mdx (hunk 1 above)
Comment (swashata) [reply src/pages/common/configs.mdx]: Got it.

**Comment by swashata** [comment]
@joshuaellis kindly let me know if the wordings are okay. English is not my first language.

**Review by joshuaellis — APPROVED** [review]
Awesome, thank you!

**Pull request merged**